src/repository/conUOpenDataCourseRepository.js

```diff
@@ -0,0 +1,120 @@
+const {createOpenDataCourse} = require("../factory/conUOpenDataCourseFactory");
+const OpenDataCourse = require('../models/conUOpenDataCourse.model');
+const axios = require("axios");
+
+module.exports = class openDataCourseRepository {
```

Comment:
Class names should start with a capital letter.

Reply:
You're right. Will fix.